superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -91,6 +91,104 @@ const StyledFilterableTable = styled.div`
   overflow-x: auto;
   margin-top: ${({ theme }) => theme.gridUnit * 2}px;
   overflow-y: hidden;
+
+  .ReactVirtualized__Grid__innerScrollContainer {
```

Comment:
Can you add the `${({ theme })}` construct at the top level to avoid repeating it for every style?